hubblestack/extmods/modules/nebula_osquery.py

```diff
@@ -318,6 +319,53 @@ def get_top_data(topfile):
     return ret
 
 
+def mask_passwords(unmasked_object):
+
+    masked_object = copy.deepcopy(unmasked_object)
+
+    try:
+        mask_file  = __salt__['cp.cache_file']('salt://mask.yaml')
```

Comment:
Can we move this into hubblestack_nebula_v2/ please? It should also be a configuration option, overridable in the config file and also via an optional argument to nebula.queries or nebula.top.